lib/coverband.rb

```diff
@@ -116,6 +116,7 @@ def self.runtime_coverage!
         start
       end
       require "coverband/integrations/resque" if defined? ::Resque
+      require "coverband/integrations/sidekiq" if defined? ::Sidekiq::Enterprise::Swarm
```

Comment:
if this is not for the free or pro, but just swarm, can the integration file be `sidekiq_swarm`